cmd/cli/kubectl-kyverno/processor/result.go

```diff
@@ -125,7 +128,7 @@ func (rc *ResultCounts) addMutateResponse(resourcePath string, response engineap
 				if mutateResponseRule.Status() == engineapi.RuleStatusPass {
 					rc.pass++
 					printMutatedRes = true
-				} else if mutateResponseRule.Status() == engineapi.RuleStatusSkip {
+				} else if mutateResponseRule.Status() == engineapi.RuleStatusSkip || mutateResponseRule.Status() == engineapi.RuleStatusNoMatch {
```

Comment:
why increment skip here ?

Reply:
Skip should only be used to indicate rules which match, but have exceptions or preconditions that skip processing.